pages/order-list/index.js

```diff
@@ -0,0 +1,3 @@
+export default function OrderList() {
+  console.log("Order List");
```

Comment:
Whenever you open a PR, make sure that you don't have commented out code or `console.log`s remaining in your code. It makes reviews a little less "cluttered" with things people don't need to look at.